runtimevar/paramstore/paramstore.go

```diff
@@ -106,8 +108,19 @@ func (w *watcher) Close() error {
 func (w *watcher) ping(_ context.Context) (*driver.Variable, error) {
 	p, err := readParam(w.sess, w.name, w.lastVersion)
 	if err != nil {
+		errCode := err.Error()
```

Comment:
Please include in your commit message what was fixed in this package.

Reply:
Done.